Makefile

```diff
@@ -64,7 +64,10 @@ else ifeq ($(FFT),fftw3_f03)
   LIBFFT=-L$(FFTW3_PATH)/lib -lfftw3 -lfftw3f
 else ifeq ($(FFT),generic)
   INC=
-  LIBFFT=#-lnvhpcwrapnvtx
+  LIBFFT=
+else ifeq ($(FFT),nvhpc)
```

Comment:
This is probably broken when FFT=nvhpc because there is no file "fft_nvhpc.f90" in decomp2d ?

Reply:
good point - totally untested code, but in principle this is how it should work, `generic` should be portable and not link NVIDIA libraries